apps/docs/src/guide/introduction/creating-a-fuel-dapp.md

```diff
@@ -0,0 +1,179 @@
+
+```md
+◇ What is the name of your project?
+│ my-fuel-project
+│
+◇ Select a package manager:
+│ pnpm
+│
+◆ Which Sway programs do you want? (space to toggle)
+│ ● Contract
+│ ○ Predicate
+│ ○ Script
+└
+```
+
+The tool will scaffold the project and install the necessary dependencies for you. You will then be greeted with this message:
+
+```md
+⚡️ Success! Created a fullstack Fuel dapp at my-fuel-project
+
+To get started:
+
+- cd into the project directory: cd my-fuel-project
+- Start a local Fuel dev server: pnpm fuels:dev
+- Run the frontend: pnpm dev
+
+-> TS SDK docs: https://docs.fuel.network/docs/fuels-ts/
+-> Sway docs: https://docs.fuel.network/docs/sway/
+-> If you have any questions, check the Fuel forum: https://forum.fuel.network/
+```
+
+## Directory Structure
+
+The project scaffolded by `npm create fuels` has roughly the following directory structure:
+
+```md
+my-fuel-project
+├── src
+│ ├── pages
+│ │ ├── index.tsx
+│ │ └── ...
+│ ├── components
+│ │ └── ...
+│ ├── styles
+│ │ └── ...
+│ └── lib.ts
+├── public
+│ └── ...
+├── sway-programs
+│ ├── contract
+│ │ └── src
+│ │ └── main.sw
+│ ├── Forc.lock
+│ └── Forc.toml
+├── fuels.config.ts
+├── package.json
+└── ...
+```
+
+It is a Next.js project with a few extra files and folders. Let's take a closer look at some of the important ones:
+
+### `./fuels.config.ts`
+
+This is the configuration file for the [`fuels` CLI](../fuels/index.md), the CLI and tooling that powers this project under the hood. It makes sure that all of your Sway programs are continuously compiled and deployed to your local Fuel node. You can read more about the `fuels.config.ts` file in the [Fuels CLI documentation](../fuels/config-file.md).
+
+### `./sway-programs/contract/src/main.sw`
+
+This is where our Sway contract lives. Out of the box, it is a simple counter contract that can only be incremented. We will add a decrement functionality to it in the next step.
+
+### `./src/pages/index.tsx`
+
+This file contains the source code for the frontend of our dApp. It is a Next.js page that renders the counter value and allows the user to increment the counter.
+
+### Dev Environment Setup
+
+Now that we have our project scaffolded, let's set up our development environment.
+
+Let's first start our Fuel Dev server. This will start a local Fuel node and continuously compile and deploy our Sway programs to it.
+
+::: code-group
+
+```sh [npm]
+pnpm fuels:dev
+```
+
+```sh [pnpm]
+pnpm fuels:dev
+```
+
+:::
+
+Once the server is up and running, we can start our Next.js development server in another terminal.
+
+::: code-group
+
+```sh [npm]
+pnpm dev
+```
+
+```sh [pnpm]
+pnpm dev
+```
+
+:::
+
+You should now be able to see the counter dApp running at `http://localhost:3000`. You can try changing the contents of the `./sway-programs/contract/src/main.sw` file and see the changes reflected in the UI without having to restart the server.
```

Comment:
```suggestion
You should now be able to see the counter dApp running at `http://localhost:4000`. You can try changing the contents of the `./sway-programs/contract/src/main.sw` file and see the changes reflected in the UI without having to restart the server.
```

Reply:
`3000` is actually correct, since that's the port where the frontend runs.

Reply:
Ah my b, 4000 was the node ofc.